src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -104,6 +104,13 @@ module sonic-flex_counter {
                 }
             }
 
+            container PG_DROP {
+                /* PG_DROP_STAT_COUNTER_FLEX_COUNTER_GROUP */
+                leaf FLEX_COUNTER_STATUS {
+                    type flex_status;
```

Comment:
do we have default value here? what is the default value?

Reply:
@lguohan @neethajohn @praveen-li  We do not have the default value here. But we can add the default "disable" to cases where the PG_DROP yang model not needed. In our case "FLEX_COUNTER_STATUS" for PG_DROP is set to enable. What do you think about this?